ext/opentelemetry-ext-django/src/opentelemetry/ext/django/middleware.py

```diff
@@ -0,0 +1,104 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from logging import getLogger
+
+from django import VERSION
+
+from opentelemetry.context import attach, detach
+from opentelemetry.ext.django.version import __version__
+from opentelemetry.ext.wsgi import (
+    collect_request_attributes,
+    get_header_from_environ,
+)
+from opentelemetry.propagators import extract
+from opentelemetry.trace import SpanKind, get_tracer
+
+if VERSION >= (1, 10, 0):
```

Comment:
All Django versions older than 2.2 are by now out of (even extended) support (they also clash with the supported python versions). So can't it just use the new method of middleware without this backwards compatibility approach?

Reply:
Removed version check